int64 a
int64 b
---
shape_msgs/MeshTriangle[] triangles
geometry_msgs/Point[] vertices
